interfaces/builtin/bluez.go

```diff
@@ -199,7 +202,15 @@ func (iface *BluezInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *i
 	case interfaces.SecurityDBus:
 		return bluezConnectedPlugDBus, nil
 	case interfaces.SecurityAppArmor:
-		return bluezConnectedPlugAppArmor, nil
```

Comment:
Hrmmm, this didn't show up in my diff before I requested the PR. Sorry about that!